data/presets/highway/crossing/_marked.json

```diff
@@ -23,11 +26,6 @@
         "key": "crossing",
         "value": "marked"
     },
-    "terms": [
```

Comment:
did you intentionally delete these `terms`? just double checking because I don't see them in any templates

Reply:
Yes. My understanding is that `terms` are ignored for unsearchable presets, so no reason to keep them around. I assume this was a copy paste error at some point.